### PR TITLE
fix(desktop): load source-map error

### PR DESCRIPTION
**Pull request by yamasite**

### CODE REVIEW

**Comment by BlackHole1** [comment]
close: https://github.com/netless-io/flat/issues/804

File: desktop/main-app/webpack/webpack.common.js [hunk 1]
```diff
@@ -83,5 +83,6 @@ module.exports = {
         filename: "[name].js",
         path: paths.dist,
         libraryTarget: "commonjs2",
+        devtoolModuleFilenameTemplate:'file:///[absolute-resource-path]',
```
Comment (BlackHole1) [inline desktop/main-app/webpack/webpack.common.js:86]: After a lot of testing, I found that this property does not play any role.

According to https://github.com/netless-io/flat/issues/804#issuecomment-915116112 I found another solution

We just need to change the devtools to `inline-source-map` in `desktop/main-app/webpack/webpack.commonjs`

e.g:
图片

Comment (BlackHole1) [inline desktop/main-app/webpack/webpack.common.js]: After a lot of testing, I found that this property does not play any role.

According to https://github.com/netless-io/flat/issues/804#issuecomment-915116112 I found another solution

We just need to change the devtools to `inline-source-map` in `desktop/main-app/webpack/webpack.commonjs`

e.g:
图片

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Comment by yamasite** [comment]
Thanks for testing. In my Mac machine, if I comment it out:

image

The error recurs:

image

If I remove the comment sign, the warnings go away:

image

Also, changing to `inline-source-map` does not remove the warning:

image

image

Could you show your outcome for `inline-source-map`?

**Comment by BlackHole1** [comment]
When I use inline-source-map, my code looks like this:

图片


I suspect it is caused by your not rebuilding. Because after using `inline-source-map`, the code should be `//# sourceMappingURL=data:application/json;charset=utf-8;base64,xxxx` and not `/# sourceMappingURL=preload.js.map`

First make sure you make the same changes as in the picture below:

图片


Then re-run `yarn run start`

This is the video I recorded:

https://user-images.githubusercontent.com/8198408/132628523-bda84cfd-0b29-4786-9541-5691873ce9bd.mp4

**Comment by yamasite** [comment]
Tried again. Works as your video indicates: 

image

Thanks.

So `devtoolModuleFilenameTemplate` does not work for you?

**Comment by yamasite** [comment]
Anyway, I think `inline-source-map` is a better approach. Have implemented the changes.

**Review by BlackHole1 — APPROVED** [review]

**Comment by BlackHole1** [comment]
Congrats on merging your first pull request! 🎉🎉🎉

**Comment by BlackHole1** [comment]
We will contact you via email in the next few days to send you a gift

**Pull request merged**